flutter/lib/desktop/widgets/scroll_wrapper.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_improved_scrolling/flutter_improved_scrolling.dart';
+
+class DesktopScrollWrapper extends StatelessWidget {
+  final ScrollController scrollController;
+  final Widget child;
+  const DesktopScrollWrapper(
+      {Key? key, required this.scrollController, required this.child})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ImprovedScrolling(
+      scrollController: scrollController,
+      enableCustomMouseWheelScrolling: false,
+      customMouseWheelScrollConfig:
+          const CustomMouseWheelScrollConfig(scrollAmountMultiplier: 3.0),
```

Comment:
3.0 is default mouse wheel scroll multiplier. Can customed to fit different platforms.

Reply:
Use constant for all these variable.

Reply:
> Use constant for all these variable.

yes, it's already done.

Reply:
and 4.0

Reply:
done

Reply:
done